projects/fsastorefrontstyles/scss/overrides/homepage.scss

```diff
@@ -65,6 +65,48 @@
     }
   }
 
+  // Need analysis entry point section
+  .Section3 {
+    .homepageQuestionnaire {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      border-radius: 2.5rem;
+      background-color: var(--cx-color-light);
+      padding: 0.5rem 0;
+      text-align: center;
+      font-size: var(--cx-h5-font-size);
+      @include media-breakpoint-down(sm) {
+        padding: 5px 0;
+      }
+      cx-generic-link a {
+        display: inline-block;
+        font-weight: var(--cx-font-weight-semi);
+      }
+      &::before,
```

Comment:
Maybe we can even use icon classes for this instead of this css?

Reply:
Where to put that icon class since I have only one cms link component?

Reply:
On the link itself? Instead of declaring `fas fa-arrow-left`, you can just put `fa-arrow-left` and then on `before` element put `font-family: "Font Awesome 5 Free";`